### PR TITLE
Docs/SK-924 | Update quickstart guide

### PR DESCRIPTION
**Pull request by ahellander**
## Description

https://fedn.readthedocs.io/en/docs-sk-924/quickstart.html

Starts implementing the Docs updates we discussed: 

  - Reorganize main menu
  - Remove local dev instructions and Docker from git-repo examples
  - Collect local dev instructions in a developer guide

### CODE REVIEW

**Review by niklastheman — COMMENTED** [review]
I have some thoughts:
- I think the "Getting started with FEDn" section should be under "Introduction".
- I think using the Studio interface should be present in the "Getting started with FEDn" guide. Then, of course we can include a short line on how to achieve the same result using the cli.
- Running the client in the "Getting started with FEDn" example is still done via docker.
- I would prefer if the "Getting started with FEDn" guide did not make use of the mnist-pytorch example but instead took the reader through creating their own FEDn project. _(perhaps could be done later)_

File: docs/quickstart.rst [hunk 1]
```diff
@@ -158,15 +189,16 @@ Access model updates
...
+You can also access global model updates via the APIClient:
 
 .. code:: python
 
    >>> ...
    >>> client.download_model("<model-id>", path="model.npz")
 
 
-**Connecting clients using Docker**
+Connecting clients using Docker
```
Comment (niklastheman) [inline docs/quickstart.rst:200]: Are we sure this is needed in the "Getting started" section?

**Review by niklastheman — COMMENTED** [review]
I think this is an improvement!

A suggestion for the later is to use tabs to display how to do things using the UI or APIClient. This is of course not for now...

File: docs/quickstart.rst [hunk 2]
```diff
@@ -7,21 +7,30 @@ Getting started with FEDn
...
 
 Set up a FEDn Studio Project
 ----------------------------
 
-Start by creating an account in FEDn Studio and set up a project by following the instruction here: :ref:`studio`.
+Start by creating an account in Studio. Head over to `fedn.scaleoutsystems.com/signup <https://fedn.scaleoutsystems.com/signup/>`_  and sign up.
+
+Logged into Studio, do: 
+
+1. Click on the "New Project" button in the top right corner of the screen.
```
[the remark this answers is not in the document]
Comment (ahellander) [reply docs/quickstart.rst:21]: Agree. Have fixed.

File: docs/quickstart.rst
[the remark this answers is not in the document]
Comment (ahellander) [reply docs/quickstart.rst:1]: True, have fixed. The plan is to rewrite this part completely to instead have the user set up the project from scratch.  But that will have to be in a separate PR.

[the remark this answers is not in the document]
Comment (ahellander) [reply docs/quickstart.rst:1]: Good, have integrated something similar.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -60,8 +56,15 @@ Next, we will prepare the client. For illustrative purposes, we use one of the p
...
-Locate into ``examples/mnist-pytorch`` folder in the cloned fedn repository. The compute package is located in the folder ``client``.
+
+To work through this quick start you need a local copy of the ``mnist-pytorch`` example project contained in the main FEDn Git repository. 
+The following command clones the entire repository but you will only use the example:
+
+.. code-block:: bash
+
+   git clone https://github.com/scaleoutsystems/fedn.git
+
+Locate into ``examples/mnist-pytorch`` folder. The compute package is located in the folder ``client``.
```
Comment (Wrede) [inline docs/quickstart.rst:67]: should be "fedn/examples/mnist-pytorch"

Comment (ahellander) [reply docs/quickstart.rst:67]: Fixed.

File: docs/quickstart.rst (hunk 1 above)
Comment (ahellander) [reply docs/quickstart.rst]: Fixed.

**Review by Wrede — APPROVED** [review]

**Pull request merged**